AIMSICD/src/main/java/com/secupwn/aimsicd/data/DefaultDataTransaction.java

```diff
@@ -0,0 +1,277 @@
+package com.secupwn.aimsicd.data;
+
+import com.secupwn.aimsicd.data.model.DefaultLocation;
+import com.secupwn.aimsicd.data.model.GpsLocation;
+import com.secupwn.aimsicd.data.model.SmsDetectionString;
+
+import io.realm.Realm;
+
+public class DefaultDataTransaction implements Realm.Transaction {
```

Comment:
Yes, a pre-populated Realm file would be a bit faster, but I see two advantages in doing it this way:

1. Its easier to change the schema
2. Its easier to see and change the default data